pyuvdata/uvdata.py

```diff
@@ -4457,11 +4457,25 @@ def get_antenna_redundancies(self, tol=1.0, include_autos=True,
         baselines in this object, this method will conjugate baselines on this
         object to the 'u>0' convention unless `no_conjugate` is set to True.
```

Comment:
This part of the docstring is out of date. There is no `no_conjugate` parameter, and I'm not sure the quoted convention is correct.